eslint.config.js

```diff
@@ -20,6 +20,8 @@ module.exports = [
       sourceType: 'script'
     },
     rules: {
+      'no-empty': 'off',
+      'no-redeclare': 'off',
       'no-var': 'off',
```

Comment:
This would be a TODO for a subsequent PR, to keep the diff small.

Reply:
And `no-var` for sure!